raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -1109,12 +1147,58 @@ def handle_lock_expired(
     return TransitionResult(mediator_state, events)
 
 
+def handle_node_change_network_state(
+        mediator_state: MediatorTransferState,
+        state_change: ActionChangeNodeNetworkState,
+        channelidentifiers_to_channels: typing.ChannelMap,
+        nodeaddresses_to_networkstates: typing.NodeNetworkStateMap,
+        pseudo_random_generator: random.Random,
+        block_number: typing.BlockNumber,
+) -> TransitionResult:
+    """ If a certain node that is part of the mediated
+        transfer route comes back online, we will try
+        to re-send the transfer.
```

Comment:
This is not what needs to be done. What we need is:

If a node comes online, and there is a channel with that node which is a route, and the route is valid, and the current transfer is stuck because there was no route available, then a transfer should be sent.